### ISSUE TITLE
Scrolling can cause posts to disappear

### ISSUE THREAD

**Issue by Kreuger** [Docile-Alligator/Infinity-For-Reddit#1241]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/issues?q=is%3Aissue+is%3Aclosed) issues to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only occurs with a certain link, post, image..., I will include the URL.

### App version

5.3.4

### Where did you get the app from

F-Droid

### Android version

11

### Device model

OnePlus Nord n10 5G

### First occurred

ongoing

### Steps to reproduce

When scrolling through posts, if you click on one and then go back, the bottom half of the app goes black. Sometimes scrolling down a few more posts will fix it and sometimes I need to force close it.

### Example post, link, markdown...

.

### Expected behaviour

Continue showing posts as normal

### Current behaviour

![Screenshot_20221119-091311](https://user-images.githubusercontent.com/42020/202855417-ddbef92c-d9ab-4218-a028-9c1e010091a2.jpg)

**Comment by scria1000** [comment]
Couldn't reproduce this. Do you have to do a lot of scrolling and back and forth before this happens to you? Does it happen on other networks? Need more info about the specific conditions.

**Comment by Kreuger** [comment]
I dont know that it requires any specific amount of time. It just happens at times. For networks, do you mean wifi/4g? Because that doesnt make a difference.

**Comment by RSBat** [comment]
Maybe the same problem https://www.reddit.com/r/Infinity_For_Reddit/comments/yb7m0c/the_subreddit_feed_is_cut_in_half/ ?

**Comment by Kreuger** [comment]
Yeah that looks the same. I wasnt sure how to word it in an attempt to search.

**Comment by RSBat** [comment]
Found different steps, happens every time (Android 12, Pixel 5):
1. Open any subreddit
2. Scroll down, then up a little (so toolbar is visible)
3. Click search. On-screen keyboard shows up
4. Click back arrow

Doesn't happen on main screen with the same steps

**Comment by Docile-Alligator** [comment]
Some libraries were updated and I think this bug is fixed. If you still encounter this problem after the next update, please reopen this issue.

**Comment by RSBat** [comment]
Still happens to me on 5.4.0-beta1 with the above steps
![Screenshot_20221204-144332.png](https://user-images.githubusercontent.com/6381474/205478220-2f317a09-a6b0-433c-a4cc-a6255aea710f.png)

**Comment by Kreuger** [comment]
I installed the beta and doing those steps doesnt cause it to happen. But I didnt try it before updating.

**Comment by Docile-Alligator** [comment]
Cannot reproduce this.

**Comment by Kreuger** [comment]
Its still happening to me as well with 5.4.0-beta4.

**Comment by Kreuger** [comment]
Still happening on 5.4.1. App is becoming unusable.

**Comment by starchyunderscore** [comment]
screen record of reproduction: 

https://youtube.com/shorts/osVetJdXUHo

(sorry its a short, youtube sucks)

5.4.2-beta1, android 12, moto g stylus 5g 2022
